go/chat/server.go

```diff
@@ -1417,155 +1427,22 @@ func (h *Server) postAttachmentLocal(ctx context.Context, arg postAttachmentArg)
 		}
 	}
 
-	// get s3 upload params from server
```

Comment:
deleting this since it is unused cc @patrickxb

Reply:
It was here for frontend for their development.  I don't think they need it anymore.

Also, @mmaxim has a proposal to not use the "in order" version, so maybe we should keep this around.

Reply:
ok i'll leave it in for now